control_panel_api/tests/test_views.py

```diff
@@ -143,13 +143,26 @@ def test_create(self, mock_aws_create_role):
 
         self.assertEqual(self.superuser.auth0_id, response.data['created_by'])
 
+    @patch('control_panel_api.models.App.aws_create_role')
```

Comment:
You are mocking but doing nothing with it.  The patch doesn't need to be there.

Reply:
👍

Reply:
ugh, I missed something :(